features/google/src/components/activitiesOverTimeStory/activitiesOverTimeStory.jsx

```diff
@@ -0,0 +1,190 @@
+                    <img
+                        src="./images/angle-right.svg"
+                        alt="arrow-right"
+                        className="space-left"
+                    />
+                </button>
+            ) : (
+                <div className="arrow filler"></div>
+            )}
+        </div>
+    );
+};
+
+export const ActivitiesOverTimeStorySummary = ({ activitiesOverTime }) => {
+    return (
+        <div className="render-summary">
+            <p className="highlighted-number">
+                {activitiesOverTime.total.toLocaleString("de-DE")}
+            </p>
+            {i18n.t("activitiesOverTimeStory:summary", {
+                number_activities: activitiesOverTime.total,
+            })}
+            <p className="source">{i18n.t("common:source.your.google.data")}</p>
+        </div>
+    );
+};
+
+export const ActivitiesOverTimeStoryDetails = ({ activitiesOverTime }) => {
+    const yearRange = fillMissingArrayValues(
+        Object.keys(activitiesOverTime.values)
+    );
+
+    const yearlyTotals = yearRange.map((year) => {
+        return {
+            title: year.toString().substring(2, 4),
+            value: activitiesOverTime.values[year]?.total || 0,
+        };
+    });
+
+    const [selectedYear, setSelectedYear] = useState(
+        yearRange[yearRange.length - 1]
+    );
+
+    const monthlyTotals = monthsAbbreviation.map((month, index) => {
+        return {
+            title: month,
+            value: activitiesOverTime.values[selectedYear]?.values[index] || 0,
+        };
+    });
+
+    const tabData = [
+        {
+            id: "total",
+            translation: i18n.t("activitiesOverTimeStory:tab.total"),
+            barData: yearlyTotals,
+            barWidth: 6,
+            barChartLegendText: i18n.t(
+                "activitiesOverTimeStory:tab.events.total",
+                {
+                    number_events: activitiesOverTime.total,
+                }
+            ),
+            barValueColor: null,
+            datePicker: (
+                <DatePicker
+                    year={selectedYear}
+                    yearRange={yearRange}
+                    onYearChange={setSelectedYear}
+                />
+            ),
+            belowChart: selectedYear,
+        },
+        {
+            id: "yearly",
+            translation: i18n.t("activitiesOverTimeStory:tab.year"),
+            barData: monthlyTotals,
+            barWidth: 22,
+            barChartLegendText: i18n.t(
+                "activitiesOverTimeStory:tab.events.yearly",
+                {
+                    number_events:
+                        activitiesOverTime.values[selectedYear]?.total || 0,
+                }
+            ),
+            barValueColor: "white",
+            datePicker: <div className="datepicker-filler" />,
+            belowChart: i18n.t("common:total.years"),
+        },
+    ];
+
+    //Adapt to order of magnitude
+    const orderOfMagnitude = (data) => {
```

Comment:
Isn't this repeated from facebookImport?

https://github.com/polypoly-eu/polyPod/blob/65d1562e9d75fff6288faf26ae5da4ef826938fe/features/facebookImport/src/components/activitiesMiniStory/activitiesMiniStory.jsx#L127-L140

Maybe not now, but can we find these pieces of code and put them somewhere, maybe in poly-look?

Reply:
Yeah all of this is copied. Putting it to poly-look would be too much work rn. We can do it step by step. I started but there's too many things. i18n, million components that would need to get extracted and tested so we get duplication